osgp/protocol-adapter-iec61850/protocol-simulator-iec61850/WebContent/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,3 @@
+Manifest-Version: 1.0

```

Comment:
Remove this file from PR (add to .gitignore?)

Reply:
Done